vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -127,15 +127,8 @@ void AdsImpl::InitializeStep4(const Result result) {
 
   NotificationAllowedCheck(false);
 
-  //ads notifications don't sustain reboot, so remove all
-  auto ads_shown_history = client_->GetAdsShownHistory();
-  if (!ads_shown_history.empty()) {
-    uint64_t ad_shown_timestamp = ads_shown_history.front();
-    uint64_t boot_timestamp = Time::NowInSeconds() - 
-        static_cast<uint64_t>(base::SysInfo::Uptime().InSeconds());    
-    if (ad_shown_timestamp <= boot_timestamp) {
-      notifications_->RemoveAll();
-    }
+  if (IsMobile()) {
```

Comment:
This business logic should only occur for Android. Please see following comment as IsMobile should be moved to that function

Reply:
excluded by preprocessor